pkg/cmd/run/shared/shared.go

```diff
@@ -42,6 +43,19 @@ type Status string
 type Conclusion string
 type Level string
 
+var RunFields = []string{
+	"name",
+	"commitMsg",
+	"createdAt",
+	"updatedAt",
+	"status",
+	"conclusion",
+	"event",
+	"id",
+	"workflowId",
```

Comment:
In `--json` output for some other commands, `id` and `somethingId` fields return GraphQL IDs because most of our data comes from GitHub API v4. However, in this case, information comes from GitHub API v3 (REST API) and so these IDs will be numeric database IDs. The different style and usability of these IDs makes me think that we should name these fields differently to avoid confusion. 🤔 

What do you think about `databaseId` and `workflowDatabaseId`?

Reply:
Makes sense 👍🏼, GitHub API v4 also refers to db IDs as `databaseId` in nodes so will go with that.